test/lib/PeopleRepoSpec.scala

```diff
@@ -13,17 +13,30 @@ class PeopleRepoSpec extends PlaySpec {
 
       implicit val demoRemoteRepo = unpackRepo("/demo-people-repos/minimal-people.git.zip")
 
-      val logins = PeopleRepo.getSponsoredUserLogins(Path.createTempDirectory(), demoRemoteRepo.getDirectory.getAbsolutePath)
-
+      val logins = PeopleRepo.getSponsoredUserLogins(Path.createTempDirectory(), demoRemoteRepo.getDirectory.getAbsolutePath, defaultBranch = "master")
       logins mustEqual Set("philwills", "tackley", "shufgy", "rtyley", "lindseydew")
     }
 
     "return an empty set, when no users.txt found" in {
 
       implicit val demoRemoteRepo = unpackRepo("/demo-people-repos/people.no-users-txt.git.zip")
 
-      val logins = PeopleRepo.getSponsoredUserLogins(Path.createTempDirectory(), demoRemoteRepo.getDirectory.getAbsolutePath)
+      val logins = PeopleRepo.getSponsoredUserLogins(Path.createTempDirectory(), demoRemoteRepo.getDirectory.getAbsolutePath, defaultBranch = "master")
+      logins mustBe empty
+    }
+
+    "extract sponsored user logins after cloning a remote repo with main default branch" in {
+
+      implicit val demoRemoteRepo = unpackRepo("/demo-people-repos/minimal-people-main.git.zip")
```

Comment:
The Travis build for this project was long-term broken before, I only just got it working now (!) - when it ran, [the tests failed](https://travis-ci.com/github/guardian/gu-who/builds/222291040#L251) as I think you've introduced a new test resource (`minimal-people-main.git.zip`) but not committed to the PR?

![image](https://user-images.githubusercontent.com/52038/113735053-c6b67e00-96f3-11eb-946f-75f599800405.png)

What's needed is a tiny zipped test repo, like https://github.com/guardian/gu-who/blob/main/test/resources/demo-people-repos/minimal-people.git.zip - but with the default branch of `main`.

Reply:
Thank you @rtyley I've pushed that file now and the build is passing.